WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderReadMoreView.swift

```diff
@@ -0,0 +1,130 @@
+import UIKit
+import SafariServices
+
+// […]
+final class ReaderReadMoreView: UIView, UIAdaptivePresentationControllerDelegate, UIPopoverPresentationControllerDelegate {
+    private let textView = UITextView.makeLabel()
+    private let infoIconView = UIImageView(image: UIImage(systemName: "info.circle"))
+    private var postURL: URL?
+
+    init(post: ReaderPost) {
+        super.init(frame: .zero)
+
+        let gradientMaskView = GradientAlphaMaskView()
+        addSubview(gradientMaskView)
+        gradientMaskView.pinEdges()
+
+        let visualEffectView = UIVisualEffectView(effect: UIBlurEffect(style: .prominent))
+        visualEffectView.layer.cornerRadius = 8
+        visualEffectView.layer.masksToBounds = true
+
+        addSubview(visualEffectView)
+        visualEffectView.pinEdges(insets: UIEdgeInsets(top: 80, left: 0, bottom: 16, right: 0))
+
+        textView.adjustsFontForContentSizeCategory = true
+
+        let string = NSMutableAttributedString(string: Strings.viewForMore, attributes: [
+            .font: UIFont.preferredFont(forTextStyle: .body),
+            .foregroundColor: UIColor.label
+        ])
+        let range = string.mutableString.range(of: "%@")
```

Comment:
I guess we'd just have to hope the `%@` in the original format doesn't get translated into `%1$@`.

Probably not for this PR, but it should not be too difficult to find out the difference between the String format and the formatted string, and compute the argument values positions in the final String.

```
func formattedString(format: String, args: [Any]) -> (final: String, argsRange: [Range])
```